spec/features/edit_a_pom_spec.rb

```diff
@@ -43,17 +45,16 @@
   end
 
   it "makes an inactive POM active", vcr: { cassette_name: 'prison_api/edit_poms_activate_pom_feature' } do
+    # This doesn't do what it appears to - the URL is wrong so we're not editing an inactive POM... :-(
```

Comment:
If this is a non-JS test, the `#inactive` will have no effect anyway.

Could we just remove it to avoid the confusing URL?